draft-ietf-rats-corim.md

```diff
@@ -1183,15 +1183,20 @@ If the search criteria are satisfied, the `endorsements` entries are asserted wi
 
 #### Conditional Endorsement Series Triple {#sec-comid-triple-cond-series}
 
-A Conditional Endorsement Series triple uses a "stateful environment" that identifies a Target Environment plus the measurements that have matching Evidence.	
+The Conditional Endorsement Series Triple is used to assert endorsed values based on an initial condition match followed by a series condition match.
+Every `series-condition-record` selection MUST select the same `mkey`s.
+Every selected `mkey`'s corresponding mentioned set of keys `mval`.key MUST be the same across each `series-condition-record`.
+For example, if a selection has 3 matching `measurement-map` statements:
 
-The series object is an array of `conditional-series-record` that has both Reference and Endorsed Values.
-Each conditional-series-record record is evaluated in the order it appears in the series array.
-The Endorsed Values are accepted if the series condition in a `conditional-series-record` matches the attester's actual state.
-The first `conditional-series-record` that successfully matches an attester's actual state terminates the matching and the corresponding Endorsed Values are accepted.
-If none of the series conditions match the attester's actual state, the triple is not matched, and no Endorsed values are accepted.	
+* where `mkey` is omitted and where `mval` equals A,
+* where `mkey` equals 1 and where `mval` equals B,
+* where `mkey` equals 2 and where `mval` equals C.
 
-More clarification about the usage and matching order will be resolved by: [^tracked-at] https://github.com/ietf-rats-wg/draft-ietf-rats-corim/issues/321
+Then all selection statements in the series will use the same 3 matching criteria.
```

Comment:
A B and C being the same across conditions would make only the first condition match if any do. You need to check that for all n > 0, for all m in cond[n].selection, there uniquely exists some m’ in cond[0].selection such that m’.mkey = m.mkey AND keys(m.mval) = keys(m’.mval).

Reply:
Likewise every mkey in cond[0] must appear in later conds.

Reply:
The existing text says that selection matching is based on the results of the condition. That means the selection conditions are not searching ACS, rather they are searching the results of the condition match. The above explanation seems to be assuming searching over ACS no?

Reply:
I made another editing pass. Is it OK?